python/sgl_jax/srt/server_args.py

```diff
@@ -1071,6 +1071,10 @@ def check_dynamic_lora_args():
                     if isinstance(self.lora_paths, dict):
                         # Dict format: {"name": "path", ...}
                         for name, path in self.lora_paths.items():
+                            if name == "0":
+                                raise ValueError(
+                                    f"0 exists in {self.lora_paths} as key, it is not allowed"
```

Comment:
Maybe add comment like: this key is a server-reserved symbol, used for requests that do not go through LoRA.